MovieDB/MovieDB/Scenes/Dashboard/DashboardUseCase.swift

```diff
@@ -0,0 +1,15 @@
+import RxCocoa
+import RxSwift
+
+protocol DashboardUseCaseType {
+    func getMoviesFromCategory(_ category: CategoryType, page: Int) -> Observable<[Movie]>
+}
+
+struct DashboardUseCase: DashboardUseCaseType {
+    private let repository = MovieRepository()
+    
+    func getMoviesFromCategory(_ category: CategoryType, page: Int) -> Observable<[Movie]> {
+        let input = CategoryMoviesEndpoint(category: category, page: page)
```

Comment:
let input = endpoint ??? 
please create new CategoryMoviesInput instead of CategoryMoviesEndpoint